home/tests/test_login.py

```diff
@@ -3,10 +3,6 @@
 
 @pytest.mark.django_db
 class TestLogin:
-    @pytest.fixture
-    def valid_user_details(self):
-        return {"username": "Lior", "password": "LiorLior"}
-
     @pytest.fixture
     def invalid_user_details(self):
         return {"username": "Lior12", "password": "LiorLior"}
```

Comment:
Maybe move this also to conftest.py so you could use it

Reply:
I have no need for this fixture in my PR (my invalid data includes more data, I did a parametrize instead)